ONSdigital/eq-survey-runner
# Eq 218 validation date

### Changes

Date type validation according to the story, which requires json changes to new structure (question types)
### How to test
1. Make sure every first works and looks right as mass design changes
2. Put in dates the same in the range and make sure an error message is generated
3. Put the from date later then the to date and make sure an error message is generated
### Who can test

Anyone but @LJBabbage
